attributes/default.rb

```diff
@@ -95,6 +95,7 @@
       '/media/downloadable' => 'restricted',
       '/pkginfo' => 'restricted',
       '/report/config.xml' => 'restricted',
+      '^/rss/(catalog/(notifystock|review)|order/new)(/|$)' => 'restricted',
```

Comment:
I've checked apache servers too and it doesn't work. I don't know why they think it'd work. Perhaps with bad server configuration only